scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py

```diff
@@ -516,14 +516,20 @@ def motr_tune_memory_config(self):
         self.logger.debug(f"FILE not found  {MOTR_M0D_CONF_FILE_PATH}\n")
         return
 
+    # Get number of services.
+    # Ex: num_of_services will be 2 since in motr services will be confd, ios
+    # But in /etc/cortx/cluster.conf io is represented by ios. So first get the service names right
```

Comment:
rephrase this comment, from the changes, we are not correcting the service names, so get service names right will be inappropriate.

Reply:
Resolved 04d9fce